torch/csrc/jit/passes/mkldnn_rewrite.cpp

```diff
@@ -13,6 +13,141 @@ namespace jit {
 
 #if AT_MKLDNN_ENABLED()
 
+namespace mkldnn {
+
+#define ATTR_MAP_ITEM(NAME)                                           \
+  {                                                                   \
+#NAME, {                                                          \
+      [](std::vector<c10::optional<at::Scalar>> scalars,              \
+         c10::optional<std::string> algorithm) {                      \
+        return ideep::attr_t::fuse_##NAME();                          \
+      },                                                              \
+          zero_scalar_operand, op_list_construct(zero_scalar_operand) \
+    }                                                                 \
+  }
+
+static constexpr float kMin = -std::numeric_limits<float>::infinity();
+static constexpr float kMax = std::numeric_limits<float>::infinity();
+
+const std::vector<std::string> zero_scalar_operand =
+    std::vector<std::string>({});
+const std::vector<std::string> one_scalar_operand =
+    std::vector<std::string>({"%alpha"});
+const std::vector<std::string> one_algorithm_operand =
+    std::vector<std::string>({"%algorithm"});
+const std::vector<std::string> two_operands =
+    std::vector<std::string>({"%alpha", "%beta"});
+
+std::string op_list_construct(std::vector<std::string> operands) {
+  std::string joined_operands = c10::Join(", ", operands);
+
+  auto rstring = at::jit::CodeTemplate(R"(
+%scalars : Scalar?[] = prim::ListConstruct(${joined_operands})
+%algorithm : str? = prim::Constant()    
+  )");
+
+  at::jit::TemplateEnv env;
+  env.s("joined_operands", joined_operands);
+
+  return rstring.format(env);
+}
+
+const std::string gelu_op_list_construct = R"(
+%scalars : Scalar?[] = prim::ListConstruct()
+)";
+
+bool aten_gelu_approximate_is_supported(
+    const Match& match,
+    const std::unordered_map<std::string, Value*>& vmap) {
+  const auto& match_vmap = match.values_map;
+  auto approximate_value =
+      graph_rewrite_helper::getIValue("algorithm", match_vmap, vmap).value();
+  return approximate_value == "none" || approximate_value == "tanh";
+}
+
+AttrFunction attr_func_none = [](std::vector<c10::optional<at::Scalar>> scalars,
+                                 c10::optional<std::string> algorithm) {
+  return ideep::attr_t();
```

Comment:
Could we make it a constant?

Reply:
Made it a `const static` variable in this function.